src/pages/library/DataBrowserDetails.js

```diff
@@ -179,8 +179,9 @@ export const SidebarComponent = ({ dataObj, id }) => {
         ])
       ]),
       h(ButtonOutline, {
-        disabled: dataObj.access !== datasetAccessTypes.GRANTED,
-        tooltip: dataObj.access === datasetAccessTypes.GRANTED ? '' : uiMessaging.controlledFeature_tooltip,
+        disabled: dataObj.access !== datasetAccessTypes.GRANTED || true,
+        tooltip: 'We are currently working on preview dataset and this will be available soon',
```

Comment:
I believe the UX guideline is to end sentences with a period (`.`).
```suggestion
        tooltip: 'We are currently working on preview dataset and this will be available soon.',
```